packages/vscode-deb/vscode-deb.pacscript

```diff
@@ -1,7 +1,7 @@
 name="vscode-deb"
 gives="code"
-version="1.76.0"
-url="http://packages.microsoft.com/repos/code/pool/main/c/code/code_${version}-1677667493_amd64.deb"
+version="1.77.3"
+url="http://packages.microsoft.com/repos/code/pool/main/c/code/code_${version}-1681292746_amd64.deb"
 description="Visual Studio Code is a lightweight but powerful source code editor which runs on your desktop and is available for Windows, macOS and Linux."
 hash="495b11f097713a355a9f9f209fdf3ca24b51576ad827d9a10810b96941ca16cc"
```

Comment:
Hey! Thanks for contributing to this project!. You need to update the `hash` here to the new one. Here are the steps:

1. Download the new `.deb` file (i.e. `url`).
2. Run `sha256sum <file>` on the downloaded file.
3. Update the `hash` here with the command's output, and commit it.

Reply:
Hello! Thank you so much for explanation, hash added